modules/sdk-core/src/bitgo/utils/abstractUtxoCoinUtil.ts

```diff
@@ -0,0 +1,7 @@
+import * as utxolib from '@bitgo/utxo-lib';
+import ScriptType2Of3 = utxolib.bitgo.outputScripts.ScriptType2Of3;
+import { isChainCode, scriptTypeForChain } from '@bitgo/utxo-lib/dist/src/bitgo';
```

Comment:
let's avoid importing from `/dist` folders. Can we instead make sure utxo-lib exports these types correctly?

Reply:
Those functions are exported under `utxo-lib/src/bitgo` so I guess we could just do `import { bitgo } from '@bitgo/utxo-lib'` and then destructure those functions from `bitgo`?